scriptworker/constants.py

```diff
@@ -354,9 +359,12 @@
                 ),
             }),
             'mobile': frozendict({
-                'focus-repo': (
+                'focus-origin-repo': (
```

Comment:
I'm not sure that adding `-origin-` doesn't make thing clearer.

Reply:
Okay. Removed.